src/jdk.management.jfr/share/classes/jdk/management/jfr/FlightRecorderMXBeanImpl.java

```diff
@@ -303,12 +303,12 @@ public void setRecordingOptions(long recording, Map<String, String> options) thr
         validateOption(ops, OPTION_DESTINATION, x -> MBeanUtils.destination(r, x));
 
         // All OK, now set them.atomically
```

Comment:
As this is a tidyup, could we remove the dot from "them.atomically" which is a typo.
I didn't see what was atomic at first.  If all options are valid, we set them all, which is great, but not really atomic.  Could just remove the word "atomically"?